.circleci/config.yml

```diff
@@ -52,20 +78,30 @@ jobs:
 
     steps:
       - checkout
+
+      # Download and cache python dependencies
       - restore_cache:
           key: deps1-{{ .Branch }}-{{ checksum "docs/requirements.txt" }}
+
       - run:
+          name: Create virtualenv
           command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install -r docs/requirements.txt
+            python3 -m venv ~/venv
+            echo "source ~/venv/bin/activate" >> $BASH_ENV
```

Comment:
I'm not familiar with CircleCI, is there a way to "factor out" the similar steps (creating a venv and activating it)?

Reply:
I'm not aware of any way to do that unfortunately :(

That would be great tho